### PR TITLE
Add ERC721BundleRentAgreement rent agreement example contract

### PR DESCRIPTION
**Pull request by peroket**

### CODE REVIEW

File: contracts/token/ERC721/ERC721BundleRentAgreement.sol [hunk 1]
```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IERC721.sol";
+import "./extensions/IERC721Rent.sol";
+import "../../utils/introspection/ERC165.sol";
+
+// Allows to manage an arbitrary number of token rentals from an arbitrary number of holder
+// contracts with only one agreement contract. The contract defines a price per time to pay
+// to rent a token, whih can be defined globally and per token. When starting a rental, the
```
Comment (ArthurBraud) [inline contracts/token/ERC721/ERC721BundleRentAgreement.sol]: which

File: test/token/ERC721/ERC721BundleRentAgreement.test.js [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+      );
+      expect(
+        await this.agreement.cancelationFeesForOwner(
+          this.token.address,
+          tokenId,
+        ),
+      ).to.bignumber.equal(defaultCancelationFee.toString());
+    });
+
+    it('cannot be cancelled by soneone else', async function () {
```
Comment (ArthurBraud) [inline test/token/ERC721/ERC721BundleRentAgreement.test.js]: someone

File: test/token/ERC721/ERC721BundleRentAgreement.test.js [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+      const tokenRental = await this.agreement.tokenRentals(
+        this.token.address,
+        tokenId,
+      );
+
+      expect(tokenRental.nextPriceInWeiPerSecond).to.bignumber.equal('1');
+      expect(tokenRental.nextCancelationFeeInWei).to.bignumber.equal('1');
+    });
+
+    it('can have the token specific cost changes by an approver', async function () {
```
Comment (ArthurBraud) [inline test/token/ERC721/ERC721BundleRentAgreement.test.js:317]: costs changed

**Review by ArthurBraud — DISMISSED** [review]

File: contracts/token/ERC721/ERC721BundleRentAgreement.sol (hunk 1 above)
Comment (peroket) [reply contracts/token/ERC721/ERC721BundleRentAgreement.sol]: done

File: test/token/ERC721/ERC721BundleRentAgreement.test.js (hunk 2 above)
Comment (peroket) [reply test/token/ERC721/ERC721BundleRentAgreement.test.js]: done

File: test/token/ERC721/ERC721BundleRentAgreement.test.js (hunk 3 above)
Comment (peroket) [reply test/token/ERC721/ERC721BundleRentAgreement.test.js:317]: done

**Review by ArthurBraud — APPROVED** [review]

**Pull request merged**